components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -534,15 +541,17 @@ export class ParsedCommand {
 export class AppCommandParser {
     private store: Store;
     private channelID: string;
+    private teamID: string;
     private rootPostID?: string;
     private intl: Intl;
 
     forms: {[location: string]: AppForm} = {};
 
-    constructor(store: Store|null, intl: Intl, channelID: string, rootPostID = '') {
+    constructor(store: Store|null, intl: Intl, channelID: string, teamID?: string, rootPostID = '') {
```

Comment:
To avoid the `|| ''` below

```suggestion
    constructor(store: Store|null, intl: Intl, channelID: string, teamID = '', rootPostID = '') {
```